src/commands/npm/dependencies/pin.ts

```diff
@@ -41,13 +41,10 @@ export default class Pin extends SfCommand<ChangedPackageVersions> {
       packageJson.writePackageJson();
     }
 
-    this.table(pkg, {
-      name: { header: 'Name' },
-      version: { header: 'Version' },
-      tag: { header: 'Tag' },
-      alias: { header: 'Alias' },
```

Comment:
note for myself: check on QA this `alias` ever existed, we didn't found it in the types.

Reply:
✅